app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java

```diff
@@ -0,0 +1,24 @@
+package com.example.parkingsystem.mvp.view;
```

Comment:
this class should be in a package "base" inside the "view" package

Reply:
Done!